### ISSUE TITLE
Question on vicuna model version

### ISSUE THREAD

**Issue by jihyukkim-nlp** [princeton-nlp/ALCE#12]
Hello, I'm wondering about the version of Vicuna that was used for the reported results in the paper.

The outcomes from my personal experiments on the ELI5 dataset show notable discrepancies from those reported in the paper.
- I used https://huggingface.co/lmsys/vicuna-13b-v1.3.
In addition, I used the full instruction, instead of light instruction.

On ELI5 dataset, 
- results from personal experiments are

| |claim recall| citation recall | citation precision|
|------|---|---|---|
|VANILLA (3-psg)|10.8|44.9|43.2|
|SUMM (10-psg)|11.7|44.4|42.6|
|SNIPPET (10-psg)|13.0|46.7|42.0|

- results from the paper:

| |claim recall| citation recall | citation precision|
|------|---|---|---|
|VANILLA (3-psg)|10.0|15.6|19.6|
|SUMM (10-psg)|4.9|9.7|12.2|
|SNIPPET (10-psg)|11.2|27.2|27.9|

I'm curious whether you used different checkpoints for Vicuna? 
I'd be grateful if you could offer any additional explanations you might have for such differences. 
While using the full instruction (rather than the light instruction) could potentially account for some differences, the gap in results appeared too substantial for that alone.

Best regards,
Jihyuk Kim

**Comment by howard-yen** [comment]
Hi Jihyuk, 

Thank you for your interest in our work. For our experiments, we used an older checkpoint of vicuna ([https://huggingface.co/lmsys/vicuna-13b-delta-v1.1](https://huggingface.co/lmsys/vicuna-13b-delta-v1.1)).

From our experience, the full instruction and light instruction do not make for significant differences; the reason that we use light instruction for vicuna is due to the limited context lengths, where using a shorter instruction while being able to fit everything else into the context is better than using a longer instruction and using the extrapolated positional embeddings.

Taking a closer look at the differences between the two version, the vicuna [repo](https://github.com/lm-sys/FastChat/blob/main/docs/vicuna_weights_version.md) states that v1.3 is trained on twice the amount of ShareGPT data than before, and 370M tokens were used for fine-tuning. I suspect that the boost in performance originates from the ShareGPT data, which we found to contain conversations similar in format to our tasks.

Thanks for bringing this to our attention!

Best,
Howard
